packages/zoe/src/types.js

```diff
@@ -52,10 +52,10 @@
  * @property {Install} install
  * @property {MakeInstance} makeInstance
  * @property {Offer} offer
- * @property {(Instance) => Object} getPublicFacet
- * @property {(Instance) => IssuerKeywordRecord} getIssuers
- * @property {(Instance) => BrandKeywordRecord} getBrands
- * @property {(Instance) => Object} getTerms
+ * @property {(instance: Instance) => Object} getPublicFacet
```

Comment:
Yeah, I was surprised the parameter name is necessary. I found this out myself only from investigating "Why am I not getting the type feedback I expected?".